jdaviz/core/template_mixin.py

```diff
@@ -2053,32 +2127,41 @@ def _subset_update(msg):
     @property
     def default_data_cls(self):
         if self.app.config == 'imviz':
-            return None
+            return CCDData
```

Comment:
CCDData does not support GWCS, so we cannot use it (see https://github.com/astropy/astropy/issues/11727).  Also, this changes app-wide behavior for Imviz. Do we really need such a drastic behavior change for batch mode in one plugin?

Might even need its own change log entry if this changes how `app.get_data` works?

```suggestion
            return NDData
```

Reply:
ok, I'll revisit this and try to see if I can work around needing to change the default here.